### PR TITLE
Fix compiler warnings

### PR DESCRIPTION
**Pull request by steviki**
First of all, thanks for creating Lottie, it is really awesome! We are using it in https://pdfviewer.io/.

As we use quite a strict configuration, a few warnings popped up for Lottie.
This PR takes care of that and fixes all of them.

Also removes the `Lottie copy-Info.plist` reference from Xcode, as this file doesn't exist.

### CODE REVIEW

File: lottie-ios/Classes/AnimatableLayers/LOTEllipseShapeLayer.m [hunk 1]
```diff
@@ -10,7 +10,7 @@
 #import "CAAnimationGroup+LOTAnimatableGroup.h"
 #import "LOTStrokeShapeLayer.h"
 
-const CGFloat kEllipseControlPointPercentage = 0.55228;
+static const CGFloat kEllipseControlPointPercentage = 0.55228;
```
Comment (kastiglione) [inline lottie-ios/Classes/AnimatableLayers/LOTEllipseShapeLayer.m]: Which compiler warning indicated the lack of static?

Comment (steviki) [inline lottie-ios/Classes/AnimatableLayers/LOTEllipseShapeLayer.m]: This was caused by `-Wmissing-variable-declarations`, saying `No previous extern declaration for non-static variable`.

File: lottie-ios/Classes/Extensions/UIColor+Expanded.m [hunk 2]
```diff
@@ -293,6 +305,12 @@ - (NSString *)LOT_stringFromColor {
 		case kCGColorSpaceModelMonochrome:
 			result = [NSString stringWithFormat:@"{%0.3f, %0.3f}", self.white, self.alpha];
 			break;
+        case kCGColorSpaceModelLab:
+        case kCGColorSpaceModelCMYK:
+        case kCGColorSpaceModelDeviceN:
+        case kCGColorSpaceModelIndexed:
+        case kCGColorSpaceModelPattern:
+        case kCGColorSpaceModelUnknown:
 		default:
```
Comment (xinsight) [inline lottie-ios/Classes/Extensions/UIColor+Expanded.m]: Just curious - what is the advantage of including all the enum cases, but leaving the `default` case? I would think that if you wanted to be explicit about the cases, then you should remove the default case. Or leave the default case, and don't both enumerating the unneeded cases.

Comment (steviki) [inline lottie-ios/Classes/Extensions/UIColor+Expanded.m]: That's right, the `default` case could be removed if all the cases are listed explicitly. I'll do that.
Thanks for the feedback!

File: lottie-ios/Classes/AnimatableLayers/LOTEllipseShapeLayer.m (hunk 1 above)
Comment (kastiglione) [inline lottie-ios/Classes/AnimatableLayers/LOTEllipseShapeLayer.m]: @steviki thanks!

**Comment by steipete** [comment]
Feedback should be addressed - let us know if you need anything else here. PDF Viewer 1.1 is now in beta with this branch of Lottie shipping - it's awesome!

**Comment by steipete** [comment]
We're just about to finish our blog post about Lottie and would love to see this merged. Can we help/update this PR? Any open questions? Thanks! Lottie is ❤️

**Comment by buba447** [comment]
@steipete Thank you for grabbing this! Sorry for the delay merging! I was out for 3 months. Im going to merge this today!

**Comment by buba447** [comment]
@steipete It looks like theres some pretty bad merge conflicts. You mind checking those out?

**Comment by steviki** [comment]
@buba447 Merge conflicts are resolved now. Should be good to go now. Let me know if you have any questions.

**Comment by ahmedgharib** [comment]
Can't wait to see this get in.

**Comment by ahmedgharib** [comment]
@buba447 when can we get this merged?

**Comment by robnadin** [comment]
Is there an update on this?

I have resolved some additional compiler warnings on my own fork but I'm reluctant to open a PR until this has been merged in first.

**Comment by buba447** [comment]
This is great! Thank you! Sorry for the delay, Ive been hard at work on V2 of Lottie!

**Comment by steviki** [comment]
Thank you @buba447!

**Pull request merged**